udata_front/theme/gouvfr/assets/js/components/AdminContentWithTooltip/AdminContentWithTooltip.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <p v-bind="$attrs" :aria-describedby="id" ref="contentRef">
+    <slot></slot>
+  </p>
+  <span class="fr-hidden-xl fr-tooltip fr-placement" :id="id" role="tooltip" aria-hidden="true">
+    {{ tooltipText }}
+  </span>
+</template>
+
+<script setup lang="ts">
+import { getRandomId } from '@datagouv/components';
+import { onMounted } from 'vue';
+import { ref } from 'vue';
+
+defineOptions({
+  inheritAttrs: false,
+});
+
+const id = getRandomId();
+const contentRef = ref<HTMLParagraphElement | null>();
+const tooltipText = ref("");
+
+onMounted(() => {
+  if(contentRef.value) {
+    tooltipText.value = contentRef.value.textContent || "";
+  }
+});
+</script>
+<style scoped>
+p {
+  line-height: var(--lh) !important;
+}
+</style>
```

Comment:
I don't understand this component at all… In the storybook I don't see any tooltip shown? You set the tooltipText from the value of the slot? For me the tooltip text should be a prop? I think I'm lost :-D

Reply:
You can hover the title column to see of it in action. The goal of the tooltip is to shown content hidden on smaller resolution. The tooltip text is computed to avoid the duplication of the content.

It allows us to call it like this, without passing `dataset.title` as props again.
```
<AdminContentWithTooltip>
    <a class="fr-link" :href="getDatasetLinkToAdmin(dataset)"> 
        {{ dataset.title }}
    </a>
</AdminContentWithTooltip>
```

Reply:
![Capture d’écran 2024-08-26 à 10 40 48](https://github.com/user-attachments/assets/e4972404-53f2-40d2-a210-db6697b3b4e2)

On small screen the title of the dataset is shown and the tooltip show the same thing…

Reply:
Yes, only if the text is small enough, you can view the other Datasets Table example below to view longer titles

![Capture d’écran 2024-08-26 à 10 50 47](https://github.com/user-attachments/assets/786080a4-e847-43e3-86c7-cbd24ca589f2)

Reply:
Ok, I understand :-) Ok for me then :-)